README.md

```diff
@@ -19,7 +19,7 @@ To learn more about Astria, please visit [astria.org](https://astria.org).
 
 * [conductor](https://github.com/astriaorg/astria/tree/main/crates/astria-conductor):
   conducts blocks from the data availability layer to the execution layer.
-* [proto](https://github.com/astriaorg/astria/tree/main/crates/astria-proto):
+* [core](https://github.com/astriaorg/astria/tree/main/crates/astria-core/src/generated):
```

Comment:
Agree with @joroshiba: this should point to the proto spec, not to a library crate (`core`), nor directly to the generated code.

There are essentially 5 components here: `conductor`, `composer`, `sequencer`, `sequencer-relayer`, and the protobuf spec.